### DISCUSSION TITLE
22.4.30

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#225] [Announcements]
Minor change to the gvmd start script for multi-container to resolve #223 

- Version updates from Greenbone.
- Component | Version | | Component | Version
----------|----------|-|----------|---------
| gvmd | v22.9.0 | | gvm_libs | v22.7.1 |
| openvas | v22.7.5 | | openvas_scanner | v22.7.5 |
| openvas_smb | v22.5.3 | | notus_scanner | v22.6.0 |
| gsa | v22.7.0 | | gsad | v22.6.0 |
| ospd | v21.4.4 | | ospd_openvas | v22.6.0 |
| pg_gvm | v22.6.1 | | python_gvm | v23.5.1 |
| gvm_tools | v23.9.0 | | greenbone_feed_sync | v23.8.0 |

And 
You might notice a change to the login screen. :)

https://github.com/sponsors/immauss

-Scott

**Comment by rezytijo** [comment]
[image](https://imgur.com/a/GeyLjzQ)

I have a problem related to installing behind a reverse proxy, after the component is run I can't log in through the reverse proxy, but when I access it without the reverse proxy it runs perfectly. The reverse proxy that I use is Traefik.

**Reply by immauss** [reply]
Take a look here. I'm not familiar with Traefik myself, but a few other users have figured it out.

https://github.com/immauss/openvas/issues/184

Please let me know if that helps or not. 

Thanks,
Scott
